cmd/start.go

```diff
@@ -25,10 +26,13 @@ func StartCommand() cli.Command {
 				Destination: &setAll,
 			},
 		},
-		Action: func() {
+		Action: func(c *cli.Context) {
+			if file.Contains(c.FlagNames(), "all") {
+				println("true")
```

Comment:
this doesn't seem to be very useful. best not to add debugging code like this (although I think you might have already removed it)

Reply:
Done